plugins/scanners/scancode/src/main/kotlin/ScanCodeResultParser.kt

```diff
@@ -143,7 +142,7 @@ private fun getLicenseFindings(result: JsonNode, parseExpressions: Boolean): Set
             LicenseFinding(
                 license = spdxLicenseExpression,
                 location = TextLocation(
-                    path = file["path"].textValue().removePrefix(input),
+                    path = file["path"].textValue(),
```

Comment:
commit: Was this a bug before? If so, should the prefix of the title be `fix!` ?

If there was something broken, it would be nice to have the impact explained.

Reply:
Nothing was broken, so it's not a bug fix. It's just superfluous to relativize paths here already as it's now being done centrally. So this is about avoiding duplicate work. See how the mentioned 6e68575eefb693307170d9b7af978260fa1b80bb does the same thing for the other scanners by removing `relativizePath()` calls. The same should have been done for ScanCode back then, too, hence the mention of a "fixup".